### PR TITLE
updated nunjucks version to 2.4.3

### PR DESCRIPTION
**Pull request by Blackweda**
I am attempting to fix the second part of:

https://github.com/mozilla/thimble.mozilla.org/issues/1715

I managed to get the wrong repository for Bramble, but now I made the changes to the correct one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
         "gunzip-maybe": "^1.3.1",
         "http-server": "^0.9.0",
         "mkdirp": "0.5.1",
-        "nunjucks": "2.3.0",
+        "nunjucks": "2.4.3",
```
Comment (gideonthomas) [inline package.json]: Same change suggested here: https://github.com/mozilla/thimble.mozilla.org/pull/1742#discussion_r102104086

**Comment by Pomax** [comment]
It looks like this PR will need a rebase before it can be applied to master again.

**Comment by humphd** [comment]
@Blackweda this is waiting on you to rebase.

**Comment by humphd** [comment]
I don't want to wait on this any longer, so I did it myself.  I've kept it at `^2.4.3` vs. `3.0.0`, and we can update again in another PR when we test that.

**Pull request merged**